src/js/pages/Arbitration/index.jsx

```diff
@@ -124,6 +125,40 @@ class Arbitration extends Component {
     }
   }
 
+  renderContactDialog() {
+    const {t, isStatus, buyerInfo, sellerInfo} = this.props;
+    const isBuyer = this.state.displayContactDialog === 'buyer';
+
+    return (
+      <ModalDialog display={!!this.state.displayContactDialog} onClose={this.displayContactDialog(false)} hideButton>
+          <RoundedIcon image={ProfileIcon} bgColor="blue" className="mb-2" />
+          {isBuyer && <Fragment>{buyerInfo.username}&apos;s <span
+            className="text-muted">{ContactMethods[stringToContact(buyerInfo.contactData).method]}</span></Fragment>}
+          {!isBuyer && <Fragment>{sellerInfo.username}&apos;s <span
+            className="text-muted">{ContactMethods[stringToContact(sellerInfo.contactData).method]}</span></Fragment>}
```

Comment:
Those two lines are theoretically in English, you need to translate them using the `<Trans>` component. You can check some of the ones I did and the docs to do that. It's a bit weird and you need some trial and error.
Let me know if you need help